pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -667,50 +667,114 @@ func (adc *attachDetachController) processVolumesInUse(
 	}
 }
 
-// installCRDs creates the specified CustomResourceDefinition for the CSIDrivers object.
-func (adc *attachDetachController) installCRDs() error {
-	crd := &apiextensionsv1beta1.CustomResourceDefinition{
+func getCSIDriverCRD() *apiextensionsv1beta1.CustomResourceDefinition {
+	return &apiextensionsv1beta1.CustomResourceDefinition{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: csiapiv1alpha1.CsiDriverResourcePlural + "." + csiapiv1alpha1.GroupName,
 		},
 		Spec: apiextensionsv1beta1.CustomResourceDefinitionSpec{
 			Group:   csiapiv1alpha1.GroupName,
 			Version: csiapiv1alpha1.SchemeGroupVersion.Version,
 			Scope:   apiextensionsv1beta1.ClusterScoped,
+			Validation: &apiextensionsv1beta1.CustomResourceValidation{
+				OpenAPIV3Schema: &apiextensionsv1beta1.JSONSchemaProps{
+					Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
+						"spec": apiextensionsv1beta1.JSONSchemaProps{
+							Description: "Specification of the CSI Driver.",
+							Properties: map[string]apiextensionsv1beta1.JSONSchemaProps{
+								"attachRequired": apiextensionsv1beta1.JSONSchemaProps{
+									Description: "Indicates this CSI volume driver requires an attach operation," +
+										" and that Kubernetes should call attach and wait for any attach operation to" +
+										" complete before proceeding to mounting.",
```

Comment:
`complete before proceeding to mounting` -> `complete before proceeding to mount`
?